### ISSUE TITLE
View tabs appear out of order with many steps

### ISSUE THREAD

**Issue by daranzolin** [daranzolin/ViewPipeSteps#7]

**Comment by joachim-gassen** [comment]
Noticed this as well. My guess is that it is triggered by View() returning prior to RStudio being done rendering the plane. I briefly looked into whether one can use the Rstudio api to sort the views but no avail. Maybe setting a small delay between the View calls could reduce the issue? But then again, it would make the addin less responsive.

**Comment by daranzolin** [comment]
Right. I did not do extensive testing, but I had previously inserted `Sys.sleep(0.02)` within the call loop, and it worked for a few cases.
